app/src/main/java/gov/hhs/cdc/trustedintermediary/context/ApplicationContext.java

```diff
@@ -11,7 +9,6 @@
 
 public class ApplicationContext {
 
-    private static volatile ApplicationContext applicationContext = null;
     private static final Map<Class<?>, Object> OBJECT_MAP = new HashMap<>();
```

Comment:
I was thinking that this HashMap will get accessed throughout many parts of the codebase. Should we make it a concurrent HashMap or maybe a synchronized one? The concurrent HashMap might give us nice optimization when adding or retrieving. What are your thoughts?

Reply:
Oh, good point.  Normally we aren't _writing_ to this `HashMap` once the application is booted up, but that's a good safety mechanism.  I'll make that change in this PR.